cylc/uiserver/handlers.py

```diff
@@ -386,6 +390,14 @@ def open(self, *args, **kwargs):  # noqa: A003
         )
 
     async def on_message(self, message):
+        with suppress(ValueError, SyntaxError):
+            message_dict = ast.literal_eval(message)
```

Comment:
We can use JSON here which should be lighter-weight as it supports fewer types:

```suggestion
            message_dict = json.loads(message)
```


(note will need to fiddle the imports)